### DISCUSSION TITLE
rtklib on apple Mac with PlayOnMac

### DISCUSSION THREAD

**Discussion by AdrianKriger** [tomojitakasu/RTKLIB#735] [Q&A]
Thank you for this amazing application... the community is pretty cool too.

Does rtklib execute successfully with [PlayOnMac](https://www.playonmac.com/en/)?

**Comment by SquirrelKnight** [comment]
To answer your question indirectly, I've got RTKlib to work with both Crossover and Wine on an M1 processor.

**Reply by AdrianKriger** [reply]
Thank you.

**Reply by valgur** [reply]
The RTKLIB Conan package also has both x64 and arm64 macOS support: https://conan.io/center/recipes/rtklib

**Reply by AdrianKriger** [reply]
Yes. Thank you. 
rtklib executes with [Whisky](https://getwhisky.app) {M2 processor} successfully too.

**Comment by AdrianKriger** [comment]
Experiencing a challenge but am not sure if it's related to the Windows emulator or perhaps something else. 
I will leave it here and see how an answer evolves.

`solution.pos` file outputs with a trailing `7` everywhere.


```text
% (lat/lon/height=WGS84/ellipsoidal,Q=1:fix,2:float,3:sbas,4:dgps,5:single,6:ppp,ns=# of satellites)
%  GPST        7 latitude(deg)7longitude(deg)7 height(m)7  Q7 ns7  sdn(m)7  sde(m)7  sdu(m)7 sdne(m)7 sdeu(m)7 sdun(m)7age(s)7 ratio7   vn(m/s)7   ve(m/s)7   vu(m/s)7     sdvn7    sdve7    sdvu7   sdvne7   sdveu7   sdvun
22997562604.0007 -33.9514327547  18.4685423267   84.31217  67  77  4.46717  3.89107 11.80947  0.81907 -2.70577  4.43817  0.007   0.07   0.000007   0.000007   0.000007  0.000007 0.000007 0.000007 0.000007 0.000007 0.00000
```

Why are these `7`'s appearing? How do I remove them / what setting does not write them please?

**Reply by SquirrelKnight** [reply]
Odd! I’ve not seen such an issue! Are you running through rtkpost (GUI) or rnx2rtkp (terminal)? What version of MacOS? Are you running through WINE or WHISKY?

You might want to try this forked version of RTKlib: https://github.com/rtklibexplorer/RTKLIB. It is updated quite regularly and has support for some newer filetypes (BSX, BIA, etc.) This is what I have been using lately, although I haven’t seen this issue with the main version either. For a bit of background, I am running the latest version of MacOS and I have run my data through both rtkpost and rnx2rtkp. I use PPP-K for my autonomous platform (no base station).

**Reply by AdrianKriger** [reply]
Gratitude for your attention. This community is pretty awesome!

My mistake.

<img src="https://github.com/tomojitakasu/RTKLIB/assets/59996720/acbe08ac-e3b8-4eb3-87ea-b1e9c31bcd4d" width="300" height="250">

`Field Separator` had a number `7`
`Blank` fixed the challenge.

Right now; I'm mainly experimenting with PPP-S and -K on high quality, survey-grade, receivers. But will hopefully move towards your, lower-cost, solution soon.
Thank you.
